### ISSUE TITLE
request of implentent

### ISSUE THREAD

**Issue by jambolo1970** [loqui/im#1110]
I wish I would do my compliments, I finally saw that it works in groups where it does not have the phone number is displayed the name and any message you have on whatsapp. Bravo :+1:  :D

first request, it would be possible to do so that you can set the display of this number in the directory or the name of the message it whatsapp?

second request, not been activated aconra the opportunity to acquire a number from a message or by a group that is not in your address book, you can avail implement it?

**Comment by Honk2** [comment]
@jambolo1970 
Your suggestions are not clear by what you have written.
Please try some other words.

**Comment by jambolo1970** [comment]
I made screenshots of what I mean by the possibility of a flag to switch between modes
[screenShot.zip](https://github.com/loqui/im/files/854308/screenShot.zip)

**Comment by Honk2** [comment]
If I understand correct, you would like to be able to change the display name in loqui and transfer a new contact from loqui to the phone's addressbook.
The working way for now is:
create a contact in the phone address book and than sync loqui/whatsapp with your address book.
This procedure would solve your first proposal.

**Comment by jambolo1970** [comment]
No, what I meant was: if it is possible to put a flag to display; if it "ON" the names in your address book; if it "OFF" the names not in address book. As from screenshot.

**Comment by Honk2** [comment]
You want to select to display the name from the addressbook or the whatsapp nick name?

**Comment by jambolo1970** [comment]
Yes! If it possible to insert this function..
